examples/chart/teleport-kube-agent/templates/secret.yaml

```diff
@@ -12,4 +11,3 @@ type: Opaque
 stringData:
   auth-token: |
     {{ coalesce .Values.joinParams.tokenName .Values.authToken }}
```

Comment:
you removed `if or .Values.authToken .Values.joinParams.tokenName` so couldn't these both be null/falsy? What gets put here in that case?

Reply:
By default, `.Values.authToken` is `""` so coalesce will write `""`.

If you override the default value with nil, it will print whatever you define